.travis.yml

```diff
@@ -9,6 +9,9 @@ env:
 script:
   - ./scripts/build-subprojects.sh
 sudo: required
+before_install:
+  - sudo add-apt-repository -y ppa:chris-lea/libsodium
+  - sudo apt install -y libsodium
```

Comment:
Try `libsodium-dev` for the package, which will pull in the runtime library that works. The runtime is probably unguessably versioned:
```bash
root@a45b83434947:/# apt-add-repository -y ppa:chris-lea/libsodium
gpg: keyring `/tmp/tmpscu9v0q4/secring.gpg' created
gpg: keyring `/tmp/tmpscu9v0q4/pubring.gpg' created
gpg: requesting key C7917B12 from hkp server keyserver.ubuntu.com
gpg: /tmp/tmpscu9v0q4/trustdb.gpg: trustdb created
gpg: key C7917B12: public key "Launchpad chrislea" imported
gpg: Total number processed: 1
gpg:               imported: 1  (RSA: 1)
OK
root@a45b83434947:/# apt-get update >/dev/null
root@a45b83434947:/# apt-get install -y libsodium-dev
Reading package lists... Done
Building dependency tree       
Reading state information... Done
The following extra packages will be installed:
  libsodium13
The following NEW packages will be installed:
  libsodium-dev libsodium13
0 upgraded, 2 newly installed, 0 to remove and 4 not upgraded.
Need to get 263 kB of archives.
After this operation, 1291 kB of additional disk space will be used.
Get:1 http://ppa.launchpad.net/chris-lea/libsodium/ubuntu/ trusty/main libsodium13 amd64 1.0.3-1chl1~trusty1 [127 kB]
Get:2 http://ppa.launchpad.net/chris-lea/libsodium/ubuntu/ trusty/main libsodium-dev amd64 1.0.3-1chl1~trusty1 [136 kB]
Fetched 263 kB in 1s (174 kB/s)       
Selecting previously unselected package libsodium13:amd64.
(Reading database ... 12917 files and directories currently installed.)
Preparing to unpack .../libsodium13_1.0.3-1chl1~trusty1_amd64.deb ...
Unpacking libsodium13:amd64 (1.0.3-1chl1~trusty1) ...
Selecting previously unselected package libsodium-dev:amd64.
Preparing to unpack .../libsodium-dev_1.0.3-1chl1~trusty1_amd64.deb ...
Unpacking libsodium-dev:amd64 (1.0.3-1chl1~trusty1) ...
Setting up libsodium13:amd64 (1.0.3-1chl1~trusty1) ...
Setting up libsodium-dev:amd64 (1.0.3-1chl1~trusty1) ...
Processing triggers for libc-bin (2.19-0ubuntu6.14) ...
```